vendor/bat-native-ads/src/bat/ads/internal/logging_util.cc

```diff
@@ -12,6 +12,23 @@
 
 namespace ads {
 
+bool ShouldAllowHeader(const std::string& header) {
```

Comment:
should we also filter response headers? they can contain sensitive info too such as the Set-Cookie header.

Reply:
nvm looks like this applies to response already

Reply:
@diracdeltas , yes, but it seems https://github.com/brave doesn't actually care about users or their privacy. What a facade Brave is; enough evidence has accumulated to dissolve that illusion for me, thank you for that.